### ISSUE TITLE
CircleLoader animation does not work in CEP7

### ISSUE THREAD

**Issue by Harbs** [adobe/spectrum-css#88]
### Expected Behavior
The CircleLoader animation should work in CEP 7 and earlier
### Actual Behavior
There is no animation
### Reproduce Scenario (including but not limited to)
http://opensource.adobe.com/spectrum-css/2.7.2/docs/#circle-loader on Chromium version 41.x
#### Steps to Reproduce
Run http://opensource.adobe.com/spectrum-css/2.7.2/docs/#circle-loader on Chromium version 41.x

#### Browser name/version/os (ie Chrome Version 62.0.3202.94 (Official Build) (64-bit) MacOS)
Chromium 41.0.2272.118
#### Spectrum-CSS version
2.7.2
#### Proposed Fix
Chromium 41.x requires -webkit- prefixes for animations.

**Comment by Harbs** [comment]
FYI, I tried writing a PR to fix this, but it seems like the fix needs to be in the autoprefixer (or browserlist) configuration. I have no experience with autoprefixer and I was not sure how to do that.

**Comment by Harbs** [comment]
FYI 2: I did test manually adding `-webkit-` and it fixed the problem.

**Comment by lazd** [comment]
@Harbs thanks for the report and the insight into the fix! It seems like the fix could be simple enough...

**Comment by Harbs** [comment]
Great. There are other similar issues, but if the autoprefixer is configured to fix this, I'm pretty sure the other issues will be fixed as well.

Once a fix is in place, I'll give another go-through to see if there are still issues...

**Comment by Harbs** [comment]
I found where to change the autoprefixer config and submitter a PR.

**Comment by GarthDB** [comment]
@larz0 could we surface a way for people to make custom builds passing in browser preferences into autoprefixer?

With the extension developers, I suspect they'll have a very specific browser targe that no one else will.

**Comment by Harbs** [comment]
@GarthDB Good idea. Requiring -webkit- prefixes for everyone is not ideal...

**Comment by Harbs** [comment]
What about a "cep7" gulp script?

**Comment by GarthDB** [comment]
That might be a better strategy than #90; I'm worried it might have unintended consequences.

**Comment by Harbs** [comment]
Also: please see my comment here: https://github.com/adobe/spectrum-css/issues/80#issuecomment-458335335

Changing the autoprefixer does not fix that problem, and manually changes get stripped out by the css processing. I'm not sure how to fix that one...

**Comment by Harbs** [comment]
I'll close #90

**Comment by GarthDB** [comment]
Changing autoprefixer should change that. Using `Chrome <= 41` on the [online autoprefixer tool](https://autoprefixer.github.io/) converts
```css
.example::placeholder {
    display: grid;
}
```
into this
```css
.example::-webkit-input-placeholder {
    display: grid;
}

.example::placeholder {
    display: grid;
}
```

**Comment by Harbs** [comment]
Interesting. Maybe it didn't work because of the `&::placeholder` instead of the explicit `.spectrum-Textfield::placeholder`?

**Comment by GarthDB** [comment]
It shouldn't:
```css
.example {
    display: grid;
    &::placeholder {
        display: absolute;
    }
}
```
outputs
```css
.example {
    display: grid;
    &::-webkit-input-placeholder {
        display: absolute;
    }
    &::placeholder {
        display: absolute;
    }
}
```

**Comment by Harbs** [comment]
I just wiped out all my temp folders and built again. This time it worked.

I was either blind the first time, or I had some temp files that was messing up the build.

Thanks!

**Comment by misterbrownlee** [comment]
This is out of scope for the browsers we support (evergreen latest and 2 versions back).  It looks like @Harbs has a work around, so let's close this for now.  Again, if we get priority from leadership we can add CEP support and re-open, but until then, let's close this.

Skipping the circle back pun this time ;)

**Comment by Harbs** [comment]
For the record, my work-around is to build from https://github.com/Harbs/spectrum-css/tree/cep-colors

In that branch I fix both the color stops and add support for Chromium 41.

For now, I'm merging main into that branch as I update.

It's an "okay" workaround as long as there's no CEP support.
